ckanext/iati_generator/blueprint/iati.py

```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint, render_template
+from ckan.plugins import toolkit
+
+iati_blueprint = Blueprint("iati_generator", __name__, url_prefix="/dataset")
```

Comment:
```suggestion
iati_blueprint = Blueprint("iati_generator", __name__, url_prefix="/iati-dataset")
```

I'll prefer not to override CKAN core URLs

Reply:
https://github.com/okfn/ckanext-iati-generator/pull/6/commits/108ef9b272d169c9417795f4298fef5e6e579428